packages/repository/test/unit/query/query-builder.unit.ts

```diff
@@ -100,10 +110,13 @@ describe('WhereBuilder', () => {
       .and({x: 'x'}, {y: {gt: 1}}, [{a: 1}, {b: 2}])
       .build();
     expect(where).to.eql({
-      a: 1,
-      b: {gt: 2},
-      c: {lt: 2},
-      and: [{x: 'x'}, {y: {gt: 1}}, {a: 1}, {b: 2}],
+      and: [
+        {a: 1, b: {gt: 2}, c: {lt: 2}},
+        {x: 'x'},
+        {y: {gt: 1}},
+        {a: 1},
+        {b: 2},
+      ],
```

Comment:
Why do we remove the ability to include multiple causes at top level, including shorthand and `and`?